### ISSUE TITLE
Firefox 13.0.1 : tabhunter no longer processes tab-closed events

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ericpromislow/tabhunter#40]
```
Open up TabHunter.

Type a pattern that hits one or more tabs

Go to one of the hits

Close that tab

The entry is still in the TabHunter list.
```

Original issue reported on code.google.com by `er...@activestate.com` on 17 Jul 2012 at 5:16

**Comment by ericpromislow** [comment]
Still not fixed with tabhunter 2.0 -- there's currently nothing watching for tab changes. If the user goes to close a tab, the popup will automatically close. So this would happen if a setTimeout handler fires and closes a tab, or creates modifications. Yeah, Tabhunter 2.0 isn't tracking that.

**Comment by ericpromislow** [comment]
not an issue anymore
